src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -677,9 +678,9 @@ citus_shard_cost_by_disk_size(PG_FUNCTION_ARGS)
 	MemoryContextSwitchTo(oldContext);
 	MemoryContextReset(localContext);
 
-	if (colocationSizeInBytes <= 0)
+	if (colocationSizeInBytes < MinRebalancerDiskSize)
```

Comment:
This setting does not quite do what I was expecting from the name. I thought it would be some minimum threshold for rebalancing.

Probably it's just a naming thing. This scenario reminds me a lot of the query plan costing settings where there's a base cost for an operation + a per byte/row cost.

Reply:
Maybe instead we should have a way of specifying a "shard base cost" instead.

Reply:
can work